app/models/follow.rb

```diff
@@ -0,0 +1,12 @@
+class Follow < ApplicationRecord
+  belongs_to :user
+  belongs_to :followable, polymorphic: true
+  enum followable_type: {Publisher: "Publisher", Author: "Author"}
```

Comment:
```suggestion
  enum followable_type: {publisher: "publisher", author: "author"}
```
mà trường followable_type để kiểu string đúng k

Reply:
cái Publisher: nó phải cùng tên với model mà nó trỏ đến ạ, 
còn followable_type là string để e tận dụng followable.class.name luôn c ạ